splipy/Curve.py

```diff
@@ -134,7 +134,7 @@ def raise_order(self, amount):
         interpolation_pts_x = N_old * self.controlpoints  # projective interpolation points (x,y,z,w)
 
         # solve the interpolation problem
-        self.controlpoints = np.linalg.solve(N_new, interpolation_pts_x)
+        self.controlpoints = np.array(np.linalg.solve(N_new, interpolation_pts_x))
```

Comment:
I tried to look up if this might happen other places, but as far as I could tell, we've been pretty consistent with always storing the controlpoints as an np.ndarray. We'll make the change here and for the time being don't bother fixing anything in the `IO/g2.py` file.